tracer/src/Datadog.Trace/HttpOverStreams/HttpHeaderHelperBase.cs

```diff
@@ -25,10 +25,5 @@ public Task WriteHeader(TextWriter writer, HttpHeaders.HttpHeader header)
         {
             return writer.WriteAsync($"{header.Name}: {header.Value}{DatadogHttpValues.CrLf}");
         }
-
-        public Task WriteEndOfHeaders(TextWriter writer)
-        {
-            return writer.WriteAsync($"Content-Type: {ContentType}{DatadogHttpValues.CrLf}{DatadogHttpValues.CrLf}");
-        }
```

Comment:
We'll need to remove all the callsites of this too.

Reply:
Ah yes, local change didn't get checked in. Done.